### PR TITLE
Observations added to View Issue action.

### PR DESCRIPTION
**Pull request by drspartez**
Included observations enable us to match performance results and underlying data.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.5.0...master
 
+### Added
```
Comment (dagguh) [inline CHANGELOG.md]: If we want to *add* this as a new contract, we should attach it to the public javadoc, e.g. in `ViewIssueAction`, describing the promised behavior.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
...
+- View Issue action updated to provide observation data together with metric (issueKey and issueId).
```
Comment (dagguh) [inline CHANGELOG.md]: Less boilerplate:
> Observe issue key and id in `ViewIssueAction`.

File: src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenarioIT.kt [hunk 3]
```diff
@@ -82,5 +83,10 @@ class JiraCoreScenarioIT {
             metric.result
         }
         Assertions.assertThat(results).containsOnly(ActionResult.OK)
+
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenarioIT.kt]: This blank line disrupts the implicit given-when-then test stages

File: CHANGELOG.md (hunk 1 above)
Comment (sebapawlak) [reply CHANGELOG.md]: AFAIK we don't want it as a _contract_. Let's delete this entry @drspartez

Comment (dagguh) [reply CHANGELOG.md]: If we don't consider it a behavioral contract then we can remove it at any point. It also means that the users can use these new observations *if* they're present, but they might go missing at any point.
We mark these as `### Fixed`.

**Pull request merged**